interview-questions.md

```diff
@@ -23,28 +23,54 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 3. Name three rails generator commands. What is created by each?
 
-  Your answer:
-
+  Your answer: 
+  generate Controller => creates a new controller in the database.
+  generate Model 
+  => Creates a new Model in rails. Models are classes and contain the data to be stored 
+  generate Migration 
+  => Allows scalability to databases by adding the ability to modify a database as well as describe the changes to the table of the database.
   Researched answer:
 
 
-
 4. Consider the Rails routes below. What is the name of the controller method that would be called by each route? What action would each of the controller methods perform?
 
-action: "GET"    route: /students          
+Index => Displays a list of all of the students
+
+action: "GET"    route: /students
 
-action: "POST"   route: /students       
+New => Returns HTML request to create a new student.
+
+action: "POST"   route: /students/create
+
+Create => Creates a new student profile.
```

Comment:
action: "GET" route: /students/new
new => displays a form to create a new student